### ISSUE TITLE
Constructions can't be applied to specific Interior Partition Surfaces in the OpenStudio Application

### ISSUE THREAD

**Issue by DavidGoldwasser** [NREL/OpenStudio#845]
You can set the construction for a construction set, but not for individual surfaces. This has to be done in the plugin. More of a feature than bug, but thought I'd document it hear.

I ran across it when trying to use one construction for interior partition surfaces that represent furniture, and another construction for interior partition surfaces that have other users.

**Comment by macumber** [comment]
@DavidGoldwasser is this about setting constructions on interior partition surfaces?  Can you clean up the text and rephrase this as a feature request?
